### ISSUE TITLE
SVG not working?

### ISSUE THREAD

**Issue by AndyVRD** [mozziemozz/home-assistant-svg-file-loader#12]
Hi,

have you any idea why [this](https://totohealth.org/assets/website/vendor/entypo/controller-volume.svg)  SVG icon is not working and/or how to get it working with your SVG loader. Thx

**Comment by mozziemozz** [comment]
Hi, no, sorry. I only tested with the ones linked in the repo.

**Comment by AndyVRD** [comment]
Ok after allot of troubleshooting i finally find the problem. The first thing, the HACS integration is not working. After checking the file-loader.js file there are important parts missing also for some reason it's not linking the /local/file-loader.js resource correctly so that's why it will never load any icons. After installing the file-loader.js manual and creating the /local/file-loader.js link correctly to the lovelace resources  it's working fine.

For anyone struggling with the correct icon name, it's "file:your_icon_name" (without the svg extension...). Also note that the svg folder needs to be in the config/www folder!

**Comment by mozziemozz** [comment]
@AndyVRD There is no HACS integration. At least not yet. I've never done that before hand haven't gotten around to study it. The readme of the repo doesn't say anything about HACS and explains how to install it manually. Filepath/name without extension is also described in the readme.

**Comment by mozziemozz** [comment]
@AndyVRD no worries. :)
This one is a fork of my repo. If i remember correctly, he contacted me regarding HACS integration. If you check his last commit comment from 2 months ago, it says "first attempt to make this HACS installable". I just saw that my repo also has a hacs.json which I added some time ago. However, it is not finished and does not currently support HACS, only manual installation. Anyway, at least you got it working manually. 🙂
